presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -44,9 +44,23 @@ in this database by running ``SHOW TABLES``::
 
     SHOW TABLES FROM mysql.web;
 
+You can see a list of the columns in the ``clicks`` table in the ``web`` database using either of the following::
+
+    describe mysql.web.clicks;
+    show columns from mysql.web.clicks;
+
 Finally, you can access the ``clicks`` table in the ``web`` database::
 
     SELECT * FROM mysql.web.clicks;
 
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``mysql`` in the above examples.
+
+MySQL Connector Limitations
+---------------------------
+
+The following SQL statements are not supported in this version of the connector::
+
+    CREATE TABLE
```

Comment:
I haven't tried this myself, but in looking at the code, CREATE TABLE should work. What error/stacktrace does this give?